examples/tour.jl

```diff
@@ -130,35 +133,38 @@ combine(groupby(signals, :recording), nrow)
 combine(s -> s[argmax(duration.(s.span)), :], groupby(signals, :recording))
 
 # Grab all multichannel signals greater than 5 minutes long:
-filter(s -> length(s.channels) > 1 && duration(s.span) > Minute(5), signals)
+subset(signals, :channels => ByRow(>(1) ∘ length), :span => ByRow(>(Minute(5)) ∘ duration))
 
 # Load all sample data for a given recording:
 target = rand(signals.recording)
-transform(view(signals, findall(==(target), signals.recording), :),
-          AsTable(:) => ByRow(load) => :samples)
+df = subset(signals, :recording => ByRow(==(target)))
+df.sample = load.(eachrow(df))
 
 # `mmap` sample data for a given LPCM signal:
-target = first(s for s in eachrow(signals) if s.file_format == "lpcm")
-Onda.mmap(target)
+i = findfirst(==("lpcm"), signals.file_format)
+Onda.mmap(signals[i, :])
 
 # Delete all sample data for a given recording (uncomment the
-# inline-commented section to actual delete filtered signals'
+# foreach line to actually delete filtered signals'
 # sample data!):
 target = rand(signals.recording)
-signals_copy = copy(signals) # we're gonna keep using `signals` afterwards, so let's work with a copy
-filter!(s -> s.recording != target #=|| (rm(s.file_path); false)=#, signals_copy)
+grps = groupby(signals, :recording)
+# foreach(rm, grps[(target,)].file_path)
+keep = DataFrame(grps[Not((target,))])
```

Comment:
```suggestion
subset!(signals_copy, [:recording, :file_path] => ByRow() do rec, path
            if rec == target
                #rm(path)  # Uncomment this line to actually delete the sample data
                return false
            end
            return true
        end)
```
This seems a bit more natural to me; I had to stare at the `groupby` + inverted index approach a bit before realizing how it worked.